wger/settings_global.py

```diff
@@ -28,6 +28,9 @@
 BASE_DIR = os.path.abspath(os.path.dirname(__file__))
 SITE_ROOT = os.path.realpath(os.path.dirname(__file__))
 
+SECRET_KEY = os.environ.get(
+    "SECRET_KEY", 'M34Ivcwrw34343Jr1hp7YzS29nVCiWqT4MgpxHSfiXNzPI5oSP8uLYXOIZ')
```

Comment:
Don't add a default. Let's force everyone to retrieve it from the environment. This way, all devs will have to set it by themselves and then we also have to update things like TravisCI and Heroku to include this environment variable.

This is that standard practice for things as sensitive as this.